### ISSUE TITLE
[feature] Move 'multi purpose' column

### ISSUE THREAD

**Issue by nailyk-fr** [tootsuite/mastodon#5745]
https://github.com/tootsuite/mastodon/pull/3207 bring ability to pin lot of columns and that is really great. 
However the 'multi purpose' column (the far-right variable column that appears for getting-started menu & searches) cannot be moved. 
While creating more than 2 custom columns, even on full HD screens, this make Mastodon difficult to use because of constant horizontal scrolling. On smaller screens like laptop it almost prevent custom columns using. 

TL;DR: request is: could it be possible to make that far-right column movable? 

P.S.: My instance is currently running the latest (2.0.0) release. 
* * * *
- [ ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

I tried looking for other issues but cannot find it. Sorry if it is a duplicate.

**Comment by Cassolotl** [comment]
I suspect this is a duplicate of #5710 - what do you think?

**Comment by nailyk-fr** [comment]
you are totally right @Cassolotl . Which one will you close? 
This one seems a bit more detailed but your use a better name. Up to you.

**Comment by Cassolotl** [comment]
I don't have the power to close yours but my observation so far is that the older one stays open and the newer one gets closed. Since both our issues are new there's not much discussion or emoji-reacting in either of them. So, er, I guess mine could stay open, since it has a better name and is older? \*shrug\*

**Comment by nailyk-fr** [comment]
Totally agree with you. Thanks for pointing the duplicate!

**Comment by Cassolotl** [comment]
@nailyk-fr Cool, and thank you for copying your commentary over! :)
